### DISCUSSION TITLE
can't jump to file from NeogitCommitView

### DISCUSSION THREAD

**Discussion by tbdoan** [NeogitOrg/neogit#1602] [Q&A] [answered]
<img width="1720" alt="image" src="https://github.com/user-attachments/assets/ca7f7faa-182c-4675-9768-f71b90e1a493" />

i'm pressing enter where the cursor is, but nothing is happening. Am on latest master (6691c4e).

**Comment by CKolkey** [comment, marked as answer]
That's never worked iirc - is there a revision that had this behavior?

**Reply by tbdoan** [reply]
i thought this pr: https://github.com/NeogitOrg/neogit/commit/991108906faf8fab4029ae421fea6cc5a6c25441

was to fix this feature

**Reply by CKolkey** [reply]
Oh, your cursor needs to be up on the filename towards the top, and it'll jump down to the hunk(s) for that file. The bugfix was just for files containing a dash character.

**Reply by tbdoan** [reply]
thanks i appreciate it

**Reply by tbdoan** [reply]
@CKolkey if i wanted to develop this feature, do you have any starting points? i'm pretty new to lua/neovim ecosystem but i really miss this workflow from jetbrains
